libcnb-test/src/container_port_mapping.rs

```diff
@@ -79,7 +79,7 @@ pub(crate) fn port_mapped_container_config(
                         (
                             format!("{port}/tcp"),
                             Some(vec![PortBinding {
-                                host_ip: None,
+                                host_ip: Some(String::from("127.0.0.1")),
```

Comment:
This will probably cause issues when the user has an IPv6 only system, which is unlikely but not impossible. It's certainly better than exposing to every interface though.